lib/rspec/core/rake_task.rb

```diff
@@ -4,9 +4,11 @@
 
 RSpec::Support.require_rspec_support "ruby_features"
 
+# :nocov:
 unless RSpec::Support.respond_to?(:require_rspec_core)
```

Comment:
When I run locally on Ruby 2.23:

> ./script/rspec_with_simplecov spec --backtrace --format progress --profile --format progress

require_rspec_core is already defined so the unless block never executes (and counts against coverage).